libs/utils/utils.js

```diff
@@ -335,13 +335,24 @@ export async function loadBlock(block) {
 
 export function decorateSVG(a) {
   const { textContent, href } = a;
-  const ext = textContent?.substr(textContent.lastIndexOf('.') + 1);
+  const altTextFlagIndex =  textContent.indexOf('|');
+  const sanitizeTextContent = altTextFlagIndex === -1
+    ? textContent
+    : textContent?.slice(0 ,altTextFlagIndex).trim();
+  const ext = sanitizeTextContent?.substring(sanitizeTextContent.lastIndexOf('.') + 1);
   if (ext !== 'svg') return;
+
+  const altText = altTextFlagIndex === -1
+    ? ''
+    : textContent.substring(textContent.indexOf('|') + 1).trim();
+  const textContentUrl = new URL(sanitizeTextContent);
+  const hrefUrl = new URL(href);
```

Comment:
`new URL` can throw errors if the url is just a slug. Are you positive that the url will always have a domain?

Reply:
Fixed. Thanks for the feedback @meganthecoder!